spec/spec_helper.rb

```diff
@@ -79,8 +79,6 @@ def loaddb
   end
 end
 
-Resque.inline = Rails.env.test?
```

Comment:
It causes an error if you don't have Resque (which is no longer required by anything else in the stack)